src/set_theory/surreal/basic.lean

```diff
@@ -92,6 +92,14 @@ end
 theorem le_of_lt {x y : pgame} (ox : numeric x) (oy : numeric y) (h : x < y) : x ≤ y :=
 not_lt.1 (lt_asymm ox oy h)
 
+theorem lt_trans {x y z : pgame} (ox : numeric x) (oy : numeric y) (h₁ : x < y)
+  (h₂ : y < z) : x < z :=
+lt_of_le_of_lt (le_of_lt ox oy h₁) h₂
+
+theorem lt_trans' {x y z : pgame} (oy : numeric y) (oz : numeric z) (h₁ : x < y)
+  (h₂ : y < z) : x < z :=
+lt_of_lt_of_le h₁ (le_of_lt oy oz h₂)
```

Comment:
It took way too long for me to work out the difference between these.
```suggestion
/-- `<` is transitive when both sides of the left inequality are numeric -/
theorem lt_trans {x y z : pgame} (ox : numeric x) (oy : numeric y) (h₁ : x < y)
  (h₂ : y < z) : x < z :=
lt_of_le_of_lt (le_of_lt ox oy h₁) h₂

/-- `<` is transitive when both sides of the right inequality are numeric -/
theorem lt_trans' {x y z : pgame} (oy : numeric y) (oz : numeric z) (h₁ : x < y)
  (h₂ : y < z) : x < z :=
lt_of_lt_of_le h₁ (le_of_lt oy oz h₂)
```
Could also consider using `_left` and `_right` in the names, up to you.